.gitlab-ci.yml

```diff
@@ -146,7 +146,7 @@ VMSMER-quality-check:
   <<: *template_quality-check
   variables:
     VIVADO_VERSION: "2019.2"
-    CLANG_TIDY_FILES: '../TestBenches/VMStubMERouter_test.cpp ../TopFunctions/CombinedConfig/VMStubMERouterTop_D1PHIA.cc'    
+    CLANG_TIDY_FILES: '../TestBenches/VMStubMERouter_test.cpp ../TopFunctions/CombinedConfig_FPGA2/VMStubMERouterTop_D1PHIA.cc'    
```

Comment:
@aryd I guess since the FPGA2 project is our focus now, we should also make this change for the ProjectionCalculator, MatchProcessor, and TrackBuilder.